doc/api/assert.md

```diff
@@ -147,6 +147,136 @@ try {
 }
 ```
 
+## Class: assert.CallTracker
+
+### `new assert.CallTracker()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Creates a new [`CallTracker`][] object which can be used to track if functions
+were called a specific number of times. The `tracker.verify()` must be called
+for the verification to take place. The usual pattern would be to call it in the
+`process` `exit` handler.
+
+```js
+const assert = require('assert');
+
+const tracker = new assert.CallTracker();
+
+function func() {}
+
+// callsfunc() must be called exactly 1 time before tracker.verify()
+const callsfunc = tracker.calls(func, 1);
+
+callsfunc();
+
+// Calls tracker.verify() and verifies if all tracker.calls() functions have
+// been called exact times.
+process.on('exit', () => {
+  tracker.verify();
+});
+```
+
+### `assert.CallTracker.calls([fn][, exact])`
```

Comment:
To conform with our other docs, shouldn't this be something like this instead?

```suggestion
### `tracker.calls([fn][, exact])`
```

See, for example usage of `buf` for instances of `Buffer` in buffer.md, or `subprocess` for instances of `ChildProcess` in child_process.md.

Reply:
Or, perhaps the highly analogous `watcher` as an instance of `FSWatcher` in fs.md.